Rancher/docs/rdt/readme.md

```diff
@@ -26,7 +28,10 @@ Windows Edition (Comming Soon...)
 - [ ] Install Fig (Optional Quality of life)
 	- [ ] Install Fig VSCode extension
 	- [ ] Set theme to dracula
+
 ### [Pull the NGINX Ingress workshop repo from GitHub](https://github.com/nginxinc/nginx-ingress-workshops/blob/main/Rancher/docs/rdt/readme.md#pull-the-nginx-ingress-workshop-repo-from-github-1)
+
+[//]: # (Is this referencing pulling through GUI? as later in the guide, we use the CLI command `git clone` to pull it down)
```

Comment:
This is just a link to the section where we initiate the git pull. I just recommend doing it via a CLI in terminal but all this part is is a link down to that section for the README.